src/functions/events/v2.ts

```diff
@@ -46,3 +42,17 @@ export type Event =
   | typeof TEST_LAB_EVENT
   | (typeof FIRESTORE_EVENTS)[number]
   | typeof FIREALERTS_EVENT;
+
+// Why can't auth context be removed? This is map was added to correct a bug where a regex
+// allowed any non-auth type to be converted to any auth type, but we should follow up for why
+// a functon can't opt into reducing PII.
+export const CONVERTABLE_EVENTS: Partial<Record<Event, Event>> = {
```

Comment:
why is this called "covertable" when we use this mapping to check whether event is not convertable?

Reply:
Because it holds the list of events that _are_ convertable. Not convertable means not in the map